service/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <name>Genome Nexus Service</name>
+    <description>Genome Nexus Service Module</description>
+
+    <artifactId>service</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0</version>
+    <parent>
+        <groupId>org.cbioportal.genome_nexus</groupId>
+        <artifactId>genome-nexus</artifactId>
+        <version>0.1.0</version>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.cbioportal.genome_nexus</groupId>
+            <artifactId>model</artifactId>
```

Comment:
Persistence already depends on this, and this module depends on Persistence, so you can remove this dependency.